usaspending_api/database_scripts/job_archive/backfill_primary_place_of_performance_scope.py

```diff
@@ -0,0 +1,174 @@
+"""
+Jira Ticket Number(s): DEV-4120
+
+    Backfill missing place_of_performance_scope values
+
+Expected CLI:
+
+    $ python3 usaspending_api/database_scripts/job_archive/backfill_primary_place_of_performance_scope.py
+
+Purpose:
+
+    Fetch place_of_performance_scope from Broker table:
+        - published_award_financial_assistance
+    And populate FABS transaction table columns:
+        - place_of_performance_scope
+"""
+import logging
+import math
+import psycopg2
+import time
+
+from os import environ
+
+
+# DEFINE THESE ENVIRONMENT VARIABLES BEFORE RUNNING!
+SPENDING_CONNECTION_STRING = environ["DATABASE_URL"]
+BROKER_CONNECTION_STRING = environ["DATA_BROKER_DATABASE_URL"]
```

Comment:
Is there a reason we aren't using `connections["broker"]` for this?

Reply:
This a standalone script in job_archive so it doesn't import the django app. (just like all the other scripts in this dir)